### DISCUSSION TITLE
add record button is disabled for models with many to many relation in prisma studio when using mongodb connector!!!

### DISCUSSION THREAD

**Discussion by TheMetaverseEngineer** [prisma/prisma#21941] [Q&A] [answered]
### Question

Just use a simple many to many relation and see for yourself. is there a reason for this or it's a bug?
I'm using 5.5.2 version

**Comment by ludralph** [comment, marked as answer]
Hi @TheMetaverseEngineer 👋 

Thank you for raising this question.

We do have an open [issue](https://github.com/prisma/studio/issues/980) for this. Please do add your use case and leave a 👍 to help our engineers prioritize it.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
